options/linux/include/sys/mount.h

```diff
@@ -13,6 +13,8 @@ extern "C" {
 #define MS_SYNCHRONOUS 32
 #define MS_NOATIME 64
 
+#define MNT_FORCE 1
```

Comment:
is this ABI?

Reply:
As far as I know it isn't (and if it is, it matches Linux now).

Reply:
this isn't consumed by the libc, so it is OS ABI

Reply:
> As far as I know it isn't (and if it is, it matches Linux now).

Arsen meant that the change should be in `abis`, not that the change is an ABI break